README.md

```diff
@@ -131,7 +132,40 @@ mars_tms = planetary_tms.get("MarsGeographicSphere")
 # Register a custom TMS (example)
 # planetary_tms.register(custom_tms)
 
-# Use with morecantile
+# Now use with morecantile
+from morecantile import Tile
+tile = Tile(x=0, y=0, z=0)
+bounds = mars_tms.bounds(tile)
+print(f"Bounds of Mars tile: {bounds}")
+```
+
+#### Using morecantile directly (without importing planetcantile)
+
+```python
+import os
+import sysconfig
+from pathlib import Path
+
+# Find planetcantile's TMS definitions directory
+site_packages = sysconfig.get_path('purelib')
+tms_dir = Path(site_packages) / "planetcantile" / "data" / "tms"
```

Comment:
right now this will fail as the TMSs are all contained in the data/v4 folder. I think we could update the project to simply use "tms" as the folder name though but it may create a large diff so I will worry about it in a follow-on pr to fix it